lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1877,6 +1899,12 @@ newtype ErrRejectedTip = ErrRejectedTip ApiNetworkTip
 showT :: Show a => a -> Text
 showT = T.pack . show
 
+instance LiftHandler ErrCurrentEpoch where
+    handler = \case
+        ErrUnableToDetermineCurrentEpoch ->
+            apiError err500 UnableToDetermineCurrentEpoch
+                "I'm unable to determine the current epoch."
```

Comment:
Why would "I" be unable to do that?
Because the node is not synced?
What can the user do about it?

Reply:
> Why would "I" be unable to do that?
> Because the node is not synced?
> What can the user do about it?

I'm pretty sure there's not much the user can do about this.

I'm going to create another PR with an enhancement to this error, once I can determine the range of conditions under which it's not possible to determine the current epoch (which is derived from the current slot).

Reply:
Issue created here: https://github.com/input-output-hk/cardano-wallet/issues/1939